README.md

```diff
@@ -30,6 +30,180 @@ For further help with using `ghasum` run:
 ghasum help
 ```
 
+## Integration
+
+To use `ghasum` in your GitHub Actions workflows:
+
+<details>
+
+<summary>For Ubuntu runners</summary>
+
+```yaml
+runs-on: ubuntu-24.04              # Also 'ubuntu-latest'
+steps:
+- name: Checkout repository        # The repository has to be checked out before
+  uses: actions/checkout@v4.2.2    #   verifying checksums because we need to
```

Comment:
Per https://github.com/chains-project/maven-lockfile/pull/1211#issuecomment-2915551658 the use of `actions/checkout` should be pinned to a commit SHA.